2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,13 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+let recipe = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"]
+};
+
+let formattedRecipe = `${recipe.title}\nServes: ${recipe.servings}\nIngredients:\n`;
```

Comment:
this is ok.
but you missed a little detail in the task description: 
`one console.log statement for each` :) 
so you could have multiple console.log statements :)